python/lsst/ap/association/association.py

```diff
@@ -98,7 +119,13 @@ def run(self,
                 nUpdatedDiaObjects=0,
                 nUnassociatedDiaObjects=0)
 
-        matchResult = self.associate_sources(diaObjects, diaSources)
+        if self.config.doTrailedSourceFilter:
+            diaTrailedResult = self.trailedSourceFilter.run(diaSources, diffIm)
+
+            matchResult = self.associate_sources(diaObjects, diaTrailedResult.diaSources)
+
```

Comment:
Why the extra newlines? There's no need to separate these.

Reply:
Removed extra lines